<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPaint">
    <property name="title">
     <string>Paint</string>
    </property>
    <addaction name="actionFreehand"/>
    <addaction name="actionCircles"/>
    <addaction name="actionSquares"/>
   </widget>
   <addaction name="menuPaint"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFreehand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Freehand</string>
   </property>
  </action>
  <action name="actionCircles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Circles</string>
   </property>
  </action>
  <action name="actionSquares">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Squares</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
